src/validation.cpp

```diff
@@ -4726,7 +4728,9 @@ bool LoadMempool(CTxMemPool& pool, CChainState& active_chainstate, FopenFn mocka
 
 bool DumpMempool(const CTxMemPool& pool, FopenFn mockable_fopen_function, bool skip_file_commit)
 {
-    int64_t start = GetTimeMicros();
+    using SteadyClock = std::chrono::steady_clock;
```

Comment:
9c832696826449236e88ff2fbb43d12d3d4ebc78 This can probably be added to util/time.h:


```diff
diff --git a/src/util/time.h b/src/util/time.h
index 0f87d66c2e..b01be93f83 100644
--- a/src/util/time.h
+++ b/src/util/time.h
@@ -24,6 +24,7 @@ struct NodeClock : public std::chrono::system_clock {
 };
 using NodeSeconds = std::chrono::time_point<NodeClock, std::chrono::seconds>;
 
+using SteadyClock = std::chrono::steady_clock;
 using SteadySeconds = std::chrono::time_point<std::chrono::steady_clock, std::chrono::seconds>;
 using SteadyMilliseconds = std::chrono::time_point<std::chrono::steady_clock, std::chrono::milliseconds>;
 using SteadyMicroseconds = std::chrono::time_point<std::chrono::steady_clock, std::chrono::microseconds>;